integration_test/Dialect/FIRRTL/abi.fir

```diff
@@ -0,0 +1,54 @@
+; RUN: rm -rf %t
+; RUN: split-file %s %t
+; RUN: firtool %t/parent.fir -o %t/parent --split-verilog
+; RUN: firtool %t/child.fir -o %t/child --split-verilog
+; RUN: verilator --lint-only -F %t/child/filelist.f -F %t/parent/filelist.f 
+
+; RUN: firtool %t/parent.fir -o %t/parent_aggs --split-verilog --scalarize-ext-modules=false --scalarize-public-modules=false --preserve-aggregate=all
+; RUN: firtool %t/child.fir -o %t/child_aggs --split-verilog --scalarize-ext-modules=false --scalarize-public-modules=false --preserve-aggregate=all
+; RUN: verilator --lint-only -F %t/child_aggs/filelist.f -F %t/parent_aggs/filelist.f 
```

Comment:
Checking this with `--default-language 1800-2017 -Wall` would be stricter and more accurate. This may require adding any necessary `-Wno-<warning>` to disable warnings we don't care about.

Also, verilator should read from stdin so this may work with single-file emission. It is also important to test multi-file emission like you are doing here.

I note that the plan for single-file emission is kind of suspect as it is not clear that this is advised/supported given the constructs that we have to emit.